package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "reactjs_mp",
+  "version": "1.0.0",
+  "main": "./src/index.js",
+  "repository": "https://github.com/tatsiana27/reactjs_mp.git",
+  "author": "Tatsiana_Kavaliova@epam.com",
+  "license": "MIT",
+  "scripts": {
+    "dev": "webpack-dev-server --env.env=dev",
```

Comment:
Try use this, please
```
"dev": "webpack-dev-server --mode development"
```

FYI: Webpack provides to us this functionality from box.